includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -317,16 +332,40 @@ private static function generate_notice_from_updates( $updates ) {
 	 * Renders a notice.
 	 *
 	 * @param array $notice See `generate_notice_from_updates` for format.
+	 * @param string $notice_id  Unique identifier for the notice.
 	 * @return void
 	 */
-	private static function render_notice( $notice ) {
-		// TODO Handle different levels of notices.
-		echo '<div class="notice wpjm-admin-notice wpjm-admin-notice--info">';
+	private static function render_notice( $notice_id, $notice ) {
+		if ( empty( $notice['actions'] ) || ! is_array( $notice['actions'] ) ) {
+			$notice['actions'] = [];
+		}
+
+		$notice_class = '';
```

Comment:
Maybe it could be an array? So we don't worry about the spacings while when adding other classes. :)
So when using, we just `implode`.

Reply:
Committed https://github.com/Automattic/WP-Job-Manager/pull/2401/commits/4cc876d2ea4b6c27f8079a8886540a29cfd44466